pkg/terminal/operator.go

```diff
@@ -42,3 +46,24 @@ func workspaceOperatorIsRunning() (bool, error) {
 	}
 	return true, nil
 }
+
+// operatorIsInstalled checks to see that a web-terminal-operator is installed on the cluster
+func operatorIsInstalled() (bool, error) {
+	config, err := rest.InClusterConfig()
+	if err != nil {
+		return false, err
+	}
+	client, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		return false, err
+	}
+
+	_, err = client.RESTClient().
+		Get().
+		AbsPath(fmt.Sprintf("/apis/%s/%s/operators/%s", operatorsGroup, operatorsVersion, webTerminalOperatorName)).
```

Comment:
After I uninstalled operator with OperatorHub on DevConsole, operator seems to live on the cluster:
```
> kc get subscription -n openshift-operators
No resources found in openshift-operators namespace.
> kc get operators
NAME                               AGE
web-terminal.openshift-operators   35m
```

Reply:
I've just pushed the changes to make it so this does a subscription check